husky_services/mediator.js

```diff
@@ -0,0 +1,34 @@
+/*
+ * This file is part of the Sulu CMS.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+define(function() {
+
+    'use strict';
+
+    var instance = null;
+
+    function Mediator() {}
```

Comment:
; is not common with this syntax. Moreover in my opinion the syntax "function Mediator()" makes more explicit that it gets used as a constructor